Packs/PAN-OS/Integrations/Panorama/README.md

```diff
@@ -1,7681 +1,4323 @@
-      </th>
-      <th>
-        <strong>Source-user</strong>
-      </th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>test</td>
-      <td>cd3f0487-3872-4691-8387-1a15e7de142b</td>
-      <td>negate: no</td>
-      <td>any</td>
-      <td>member: any</td>
-      <td>ssl-forward-proxy: null</td>
-      <td>any</td>
-      <td>any</td>
-      <td>any</td>
-      <td>any</td>
-      <td>no-decrypt</td>
-      <td>any</td>
-    </tr>
-  </tbody>
-</table>
-</p>
-
-<h3 id="panorama-get-wildfire-configuration">81. panorama-get-wildfire-configuration</h3>
-<hr>
-<p>Retrieve Wildfire Configuration</p>
-<h5>Base Command</h5>
-<p>
-  <code>panorama-get-wildfire-configuration</code>
-</p>
-
-<h5>Input</h5>
-<table style="width:750px" border="2" cellpadding="6">
-  <thead>
-    <tr>
-      <th>
-        <strong>Argument Name</strong>
-      </th>
-      <th>
-        <strong>Description</strong>
-      </th>
-      <th>
-        <strong>Required</strong>
-      </th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>template</td>
-      <td>The template name.</td>
-      <td>Required</td>
-    </tr>
-  </tbody>
-</table>
-
-<p>&nbsp;</p>
-<h5>Context Output</h5>
-<table style="width:750px" border="2" cellpadding="6">
-  <thead>
-    <tr>
-      <th>
-        <strong>Path</strong>
-      </th>
-      <th>
-        <strong>Type</strong>
-      </th>
-      <th>
-        <strong>Description</strong>
-      </th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>Panorama.WildFire.Name</td>
-      <td>String</td>
-      <td>File type.</td>
-    </tr>
-    <tr>
-      <td>Panorama.WildFire.Size-limit</td>
-      <td>String</td>
-      <td>File size limit.</td>
-    </tr>
-    <tr>
-      <td>Panorama.WildFire.recurring</td>
-      <td>String</td>
-      <td>Schedule recurring</td>
-    </tr>
-  </tbody>
-</table>
-
-<p>&nbsp;</p>
-<h5>Command Example</h5>
-<p>
-  <code>!panorama-get-wildfire-configuration template=WildFire</code>
-</p>
-<h5>Context Example</h5>
-<pre>
+This integration supports both Palo Alto Networks Panorama and Palo Alto Networks Firewall. You can create separate instances of each integration, and they are not necessarily related or dependent on one another.
+Manage Palo Alto Networks Firewall and Panorama. For more information see Panorama documentation.
+This integration was integrated and tested with version 8.1.0 of Palo Alto Firewall, Palo Alto Panorama
+
+## Panorama Playbook
+* **PanoramaCommitConfiguration** : Based on the playbook input, the Playbook will commit the configuration to Palo Alto Firewall, or push the configuration from Panorama to predefined device groups of firewalls. The integration is available from Demisto v3.0, but playbook uses the GenericPooling sub-playbook, which is only available from Demisto v4.0.
+* **(Deprecated) PanoramaQueryTrafficLogs** : Use the Panorama Query Logs playbook instead.W raps the following commands with genericPolling to enable a complete flow to query traffic logs.
```

Comment:
this is the new `Panorama Query Logs`, so can remove.

Reply:
@bakatzir - your are talking about PanoramaQueryTrafficLogs specifically right?